src/components/Home/Home.js

```diff
@@ -17,27 +17,27 @@ import SearchIcon from "@material-ui/icons/Search";
 import "./Home.css";
 import VaccineDataMain from "../VaccineData/VaccineDataMain";
 import Pagination from "../Pagination/Pagination";
+import { useDataLayerValue } from "../../Context/DataLayer";
 
 const Home = () => {
-  const [state, setState] = useState([]);
-  const [stateCode, setStateCode] = useState("States");
-  const [districts, setDistricts] = useState([]);
-  const [districtCode, setDistrictCode] = useState(
-    "PLEASE SELECT A STATE FIRST"
-  );
+  const [
+    {
+      states,
+      districts,
+      vaccineData,
+      toSearch,
+      toSearchValue,
+      stateCode,
+      districtCode,
+      vaccinePerPage,
+      currentPage,
+    },
+    dispatch,
+  ] = useDataLayerValue();
+
   const [pin, setPin] = useState("");
   const [formattedDate, setFormattedDate] = useState("");
   const [selectedDate, setSelectedDate] = useState(new Date());
```

Comment:
@Justinnn07 why are the above states not used under context?

Reply:
These states most prob. won't be used outside these components , so thought of maintaining as it is! 

Well, if you want i can specify them under context